plugins/legacy-preset-chart-big-number/src/BigNumber/transformProps.ts

```diff
@@ -146,7 +147,9 @@ export default function transformProps(chartProps: BigNumberChartProps) {
     });
   }
 
-  const formatNumber = getNumberFormatter(yAxisFormat);
+  const headerFormatter = headerFormatSelector
```

Comment:
Can we rename `headerFormatSelector` to `isFormatTimestamp` or something like that? It's not obvious that it's a boolean value which is true if formatter is for timestamp

Reply:
I will push a separate PR after this. Thanks!